primeqa/qg/requirements.txt

```diff
@@ -0,0 +1,14 @@
+# these are gpu specific so installed seperately
+torch==1.3.0
+transformers==3.3.1
+ipython
+nltk
+tabulate==0.8.7
+tqdm==4.59.0
+pandas
+babel==2.9.0
+records
+jedi==0.17.2
+flask
+cowsay
+sentencepiece
```

Comment:
Is requirements.txt needed ? Or is it integrated into setup.py with consistent versions.

Reply:
it is consistent with setup.py, hence removed.